server/services/memory/migration_task.py

```
@@ -0,0 +1,157 @@
+                            if row:
+                                create_version(cursor, str(row[0]), org_id, user_id, part_content, source="migration")
+                                stats["documents"] += 1
+                    except Exception as e:
+                        logger.warning(f"[Migration] Error migrating doc {filename} for org {org_id}: {e}")
+                        stats["errors"] += 1
+                        conn.rollback()
+
+                conn.commit()
+
+    except Exception as e:
+        logger.error(f"[Migration] Fatal error: {e}")
```

Ruff (0.15.18)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)